ray_on_golem/reputation/updater.py

```diff
@@ -0,0 +1,84 @@
+from contextlib import contextmanager
+from typing import Iterable, Tuple
+
+import aiohttp
+
+from ray_on_golem.reputation import models as m
+
+REPUTATION_SYSTEM_URI = "https://reputation.dev-test.golem.network/v1/"
+REPUTATION_SYSTEM_PROVIDER_SCORES = "providers/scores"
+
+
+class ReputationUpdaterException(Exception):
```

Comment:
```suggestion
class ReputationUpdaterException(RayOnGolemError):
```
We are keeping all app-related exceptions on top of `RayOnGolemError`.